docs/potts.rst

```diff
@@ -104,8 +116,8 @@ Potts section:
          …
     </Potts>
 
-By default we use ``Min`` function. Notice, that if you use global
-fluctuation amplitude definition ``Temperature`` it does not really matter
+By default, we use the ``Min`` function. Notice that if you use the global
+fluctuation amplitude definition ``Temperature``, it does not really matter
 which function you use. The differences arise when *"source"* and
 *"destination"* cells have different fluctuation amplitudes.
```

Comment:
I want to delete this and the code below about ``FluctuationAmplitudeFunctionName`` or move it to a different doc. I feel like need two versions of ``potts.rst``: one introductory version and one comprehensive version.